app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/medication/MedicationDetailsViewModel.kt

```diff
@@ -143,6 +146,23 @@ class MedicationDetailsViewModel @Inject constructor(
         return medicationDetails
     }
 
+    private fun getAddress(dispensingPharmacy: DispensingPharmacyDto): String {
+        var address = ""
+        if (!dispensingPharmacy.addressLine1.isNullOrBlank()) {
+            address = address.plus(dispensingPharmacy.addressLine1)
```

Comment:
Instead of adding if conditions, you can write like this
`address = address.plus(dispensingPharmacy.addressLine1 ?: "")`